wp-includes/rest-api/endpoints/class-wp-rest-customize-changesets-controller.php

```diff
@@ -94,4 +133,835 @@ public function register_routes() {
+			'type'              => 'array',
+			'items'             => array(
+				'enum'          => array_merge( $this->statuses, array( 'any' ) ),
+				'type'          => 'string',
+			),
+			'sanitize_callback' => array( $this, 'sanitize_post_statuses' ),
+			'default'           => array( 'auto-draft' ),
+		);
+
+		$query_params['offset'] = array(
+			'description'        => __( 'Offset the result set by a specific number of items.' ),
+			'type'               => 'integer',
+		);
+
+		$query_params['order'] = array(
+			'description'        => __( 'Order sort attribute ascending or descending.' ),
+			'type'               => 'string',
+			'default'            => 'desc',
+			'enum'               => array( 'asc', 'desc' ),
+		);
+
+		$query_params['orderby'] = array(
+			'description'        => __( 'Sort collection by object attribute.' ),
+			'type'               => 'string',
+			'default'            => 'date',
+			'enum'               => array(
+				'date',
+				'relevance',
+				'id',
+				'title',
+				'slug',
+			),
+		);
+
+		/**
+		 * Filter collection parameters for the customize_changesets controller.
+		 *
+		 * This filter registers the collection parameter, but does not map the
+		 * collection parameter to an internal WP_Query parameter.
+		 *
+		 * @since 4.?.?
+		 *
+		 * @param array $query_params JSON Schema-formatted collection parameters.
+		 */
+		return apply_filters( "rest_{$this->post_type}_collection_params", $query_params );
+	}
+
+	/**
+	 * Prepares a single customize changeset post output for response.
+	 *
+	 * @since 4.7.0
+	 * @access public
+	 *
+	 * @param WP_Post         $changeset_post    Customize changeset object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $changeset_post, $request ) {
+
+		$manager = $this->ensure_customize_manager( $changeset_post->post_name );
+
+		$data = array();
+
+		$data['date'] = $this->prepare_date_response( $changeset_post->post_date_gmt, $changeset_post->post_date );
+		if ( '0000-00-00 00:00:00' === $changeset_post->post_date_gmt ) {
+			$post_date_gmt = get_gmt_from_date( $changeset_post->post_date );
+		} else {
+			$post_date_gmt = $changeset_post->post_date_gmt;
+		}
+		$data['date_gmt'] = $this->prepare_date_response( $post_date_gmt );
+
+		$data['slug'] = $changeset_post->post_name;
+		$data['status'] = $changeset_post->post_status;
+
+		$data['title'] = array(
+			'raw'      => $changeset_post->post_title,
+			'rendered' => get_the_title( $changeset_post->ID ),
+		);
+
+		$raw_settings = json_decode( $changeset_post->post_content, true );
+
+		$settings = array();
+		if ( is_array( $raw_settings ) ) {
+			foreach ( $raw_settings as $setting_id => $params ) {
+
+				$setting = $manager->get_setting( $setting_id );
+				if ( ! $setting || ! $setting->check_capabilities() ) {
+					continue;
+				}
+				$settings[ $setting_id ] = array(
+					'value' => $params['value'],
```

Comment:
The other params should pass through. So I think this can just be `$settings[ $setting_id ] = $params`